ci/Jenkinsfile

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env groovy
+library 'status-jenkins-lib@v1.7.13'
+
+pipeline {
+    agent {
+        label 'linux && x86_64 && qt-5.15.2'
+    }
+
+    environment {
+        SQUISH_DIR = "/opt/squish-runner-7.1-20230222-1555"
```

Comment:
Hardcoding this is a bad idea. You shoudl just use the symlink:
```
admin@linux-01.he-eu-hel1.ci.devel:~ % ls -ld /opt/squish-runner*
lrwxrwxrwx  1 root root   36 Jun 20 13:05 /opt/squish-runner -> /opt/squish-runner-7.1-20230222-1555
drwxr-xr-x 16 root root 4096 Jun 20 18:31 /opt/squish-runner-7.1-20230222-1555
```
Unless you've hardcoded this intentionally?

Reply:
I would call it not Hard Coding but a Constant variable that same for all machines.

Reply:
It's hardcoding an specific path. That's what hardcoding is.